app/models/enterprise.rb

```diff
@@ -251,12 +251,21 @@ def self.distinct_count
 
   # Remove any unsupported HTML.
   def long_description
-    HtmlSanitizer.sanitize(super)
+    HtmlSanitizer.sanitize_and_enforce_link_target_blank(super)
```

Comment:
I introduced a `sanitize_and_enforce_link_target_blank` method because I wasn't sure if  `target="_blank"`  is needed _all_ the time, or just _some_ of the time.

One other thing I wasn't sure about was whether to use [HtmlSanitizer](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/services/html_sanitizer.rb) or [TrixSanitizer](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/services/trix_sanitizer.rb), they both seem similar but they are not the exact same.

Reply:
From memory `HtmlSanitizer` was introduced specifically for `Enterprise` model so this is the correct place to add `sanitize_and_enforce_link_target_blank`.

Reply:
Sounds good to me. I'm guessing that the previous solution did the same thing? (enforce link `target="blank"` on _all_ links). 
I'm guessing it was because the links would normally point to an external website. It raises the question what if people use internal links to the current website. But that should be answered in a separate issue, if needed.

Reply:
Yes, previously the text angular editor would add a `target="_blank"` attribute every time you inserted a link, see https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/assets/javascripts/admin/utils/directives/textangular_links_target_blank.js.coffee